examples/simple/.envrc

```diff
@@ -1,4 +1,8 @@
-watch_file devenv.nix 
-watch_file devenv.yaml 
-watch_file devenv.lock
-eval "$(devenv print-dev-env)"
\ No newline at end of file
+if ! has nix_direnv_version || ! nix_direnv_version 2.2.0; then
+  source_url "https://raw.githubusercontent.com/nix-community/nix-direnv/2.2.0/direnvrc" "sha256-5EwyKnkJNQeXrRkYbwwRBcXbibosCJqyIUuz9Xq+LRc="
+fi
+
```

Comment:
Should we here add something like `$(which devenv)/bin` to $PATH?

Reply:
At this point, `devenv` is not on the PATH yet, if you assume the user hasn't installed it on their machine. So that would actually fail.